tools/devtools/src/devtools_app/devtools_window/components_tab/details_window/object_tree_element/object_tree_element.js

```diff
@@ -0,0 +1,115 @@
+import { evalInWindow } from "../../../../../utils";
+import { useStore } from "../../../../store/store";
+
+const { Component, useState, onWillUpdateProps, useEffect } = owl;
+
+export class ObjectTreeElement extends Component {
+  static props = [
+    "name",
+    "content",
+    "children",
+    "display",
+    "toggled",
+    "depth",
+    "contentType",
+    "hasChildren",
+    "objectType",
+    "editReactiveState",
+    "updateBag",
+    "componentPath",
+    "loadGetterContent",
+  ];
+
+  static template = "devtools.ObjectTreeElement";
+
+  static components = { ObjectTreeElement };
+
+  setup() {
+    this.state = useState({
+      editMode: false,
+    });
+    this.store = useStore();
+    useEffect(
+      (editMode) => {
+        // Focus on the input when it is created
+        if (editMode) {
+          const input = document.getElementById("objectEditionInput/" + this.pathAsString);
+          input.focus();
+          input.select();
+        }
+      },
+      () => [this.state.editMode]
+    );
+  }
+
+  get pathAsString() {
+    return JSON.stringify(this.props.path);
+  }
+
+  get objectName() {
+    if (this.props.contentType === "function" && this.props.content.startsWith("get "))
+      return "get " + this.props.name;
+    return this.props.name;
+  }
+
+  setupEditMode(ev) {
+    if (!this.state.editMode) {
+      if (["number", "string", "boolean", "undefined"].includes(this.props.contentType)) {
+        this.state.editMode = true;
+      }
+    }
+  }
+
+  editObject(ev) {
+    let value = ev.target.value;
+    if (ev.keyCode === 13 && value !== "") {
+      console.log(value);
+      if (this.props.contentType === "string") {
+        value = JSON.stringify(value.substring(1, value.length - 1));
+      }
```

Comment:
Changed this to be more intuitive on edition (remove "" from strings when in edit mode). The JSON.stringify(value) must remain though because else it will be passed as an unknown variable name instead of a value in the eval afterwards